src/ts/utils/roam.ts

```diff
@@ -43,6 +44,10 @@ export const Roam = {
         return Promise.reject('We\'re currently not inside roam block');
     },
 
+    async editBlock(element: HTMLElement) {
```

Comment:
`makeBlockEditable` maybe. Also should it check that it got an appropriate element?